server/src/test/java/com/tencent/rss/server/StorageCheckerTest.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Tencent is pleased to support the open source community by making
+ * Firestorm-Spark remote shuffle server available.
+ * Copyright (C) 2021 THL A29 Limited, a Tencent company.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * https://opensource.org/licenses/Apache-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.tencent.rss.server;
+
+import org.junit.Test;
+
+import java.io.File;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class StorageCheckerTest {
+
+  private int mode = 0;
+
+  @Test
+  public void checkTest() throws Exception {
+    ShuffleServerConf conf = new ShuffleServerConf();
+    conf.set(ShuffleServerConf.RSS_STORAGE_BASE_PATH, "st1,st2,st3");
+    conf.set(ShuffleServerConf.RSS_HEALTH_MIN_STORAGE_PERCENTAGE, 55.0);
+    StorageChecker checker = new MockStorageChecker(conf);
+
+    assertTrue(checker.checkIsHealthy());
+
+    mode++;
+    assertTrue(checker.checkIsHealthy());
+
+    mode++;
+    assertFalse(checker.checkIsHealthy());
+
+    mode++;
+    assertTrue(checker.checkIsHealthy());
+    conf.set(ShuffleServerConf.RSS_HEALTH_MIN_STORAGE_PERCENTAGE, 80.0);
+    checker = new MockStorageChecker(conf);
+    assertFalse(checker.checkIsHealthy());
+
+    mode++;
+    checker.checkIsHealthy();
+    assertTrue(checker.checkIsHealthy());
+  }
+
+  private class MockStorageChecker extends StorageChecker {
+    public MockStorageChecker(ShuffleServerConf conf) {
+      super(conf);
+    }
+
+    @Override
+    long getTotalSpace(File file) {
+      return 1000;
+    }
+
+    @Override
+    long getUsedSpace(File file) {
+      long result = 0;
+      switch (file.getPath()) {
+        case "st1":
+          switch (mode) {
```

Comment:
what doese this *mode* mean?

Reply:
This is test.  Just the method mode.

Reply:
This is simple number, could you add some comments since other people may need to read or add your ut.